### PR TITLE
nautilus: tools/rados: Unmask '-o' to restore original behaviour

### PR DESCRIPTION
**Pull request by tchaikov**
https://tracker.ceph.com/issues/44360

### CODE REVIEW

**Comment by badone** [comment]
jenkins test make check

**Comment by badone** [comment]
```
The following tests FAILED:
	  7 - run-tox-mgr-ansible (Failed)
	  8 - run-tox-mgr-orchestrator_cli (Failed)

`Traceback:
test_orchestrator.py:5: in <module>
    from orchestrator import DriveGroupSpec, DeviceSelection, DriveGroupValidationError, \
../orchestrator.py:15: in <module>
    from mgr_module import MgrModule, PersistentStoreDict
../mgr_module.py:13: in <module>
    import rados
E   ImportError: No module named rados`
```

**Comment by tchaikov** [comment]
jenkins test make check

i cannot reproduce this failure locally.

**Comment by tchaikov** [comment]
jenkins test make check

**Comment by badone** [comment]
I can reproduce on latest bionic container image. Looking into it further.

**Comment by badone** [comment]
Looks like multiple Nautilus PRs are hitting this issue. I'm trying to bisect it but I have no reason it's related to this code.

File: PendingReleaseNotes [hunk 1]
```diff
@@ -43,3 +43,8 @@
   Alternatively, the warning can be silenced with::
 
     ceph config set global mon_warn_on_pool_pg_num_not_power_of_two false
+
+* The behaviour of the ``-o`` argument to the rados tool has been reverted to
+  its orignal behaviour of indicating an output file. This reverts it to a more
+  consisten behaviour when compared to other tools. Specifying obect size is now
```
Comment (runsisi) [inline PendingReleaseNotes]: s/obect/object/

Comment (tchaikov) [reply PendingReleaseNotes]: s/consisten/consistent/

we might want to fix these typos in master first though.

**Comment by liewegas** [comment]
see fb991fa5f62c4aaa6c564a6d15aeee4bfaa33b39

**Comment by smithfarm** [comment]
@tchaikov Please rebase. This is targeting 14.2.9 now.

**Comment by tchaikov** [comment]
@smithfarm thanks for the reminder. rebased and pushed.

**Comment by yuriw** [comment]
@tchaikov pls rebase
Auto-merging qa/tasks/radosbench.py
Auto-merging PendingReleaseNotes
CONFLICT (content): Merge conflict in PendingReleaseNotes
Automatic merge failed; fix conflicts and then commit the result.
Traceback (most recent call last):
  File "/home/yuriw/wip_master/src/script/build-integration-branch", line 100, in <module>
    assert not r
AssertionError

**Comment by tchaikov** [comment]
@yuriw rebased

**Comment by smithfarm** [comment]
jenkins test make check

**Comment by yuriw** [comment]
[wip-yuri-testing-2020-04-01-2252-nautilu](https://trello.com/c/86sLwe7i/859-wip-yuri-testing-2020-04-01-2252-nautilu)

**Pull request merged**